### DISCUSSION TITLE
Social network

### DISCUSSION THREAD

**Discussion by Vort** [community/community#37444] [Other Feature Feedback, Questions, & Ideas]
I "collaborated" with someone and now I see all his actions on main GitHub page.
This is wrong. If I want to follow someone, then... I follow him.
If I'm interested in some project, I will star it and watch it.

It may be interesting feature in case someone want to explore.
But such desire happens not for everyone and not every time.

**Comment by tomaarsen** [comment]
Agreed. I want to use the homepage like I always have: to track my projects of interest and observe follows and stars that my work receives. I could care less what other people that I don't follow are up to.

**Comment by ossanna16** [comment]
Hi there @Vort and welcome to our community! Thank you for the feedback 🙂

**Comment by superaffectivejosh** [comment]
Thank you :)
